SSZipArchive/minizip/mz_crypt_apple.c

```diff
@@ -6,9 +6,13 @@
 
    This program is distributed under the terms of the same license as zlib.
    See the accompanying LICENSE file for the full text of the license.
+
+WARNING: Be very careful updating/overwriting this file.
+It has specific changes for SSZipArchive support for Mac App Store
```

Comment:
This additional warning is just to say that I cherry-picked https://github.com/zlib-ng/minizip-ng/pull/804 to avoid App Store rejection. We can remove it once that PR is merged upstream.

Reply:
upstream PR was merged, so I've removed that warning